out.txt

```diff
@@ -0,0 +1,105 @@
+Microsoft (R) Build Engine version 15.7.179.6572 for .NET Framework
+Copyright (C) Microsoft Corporation. All rights reserved.
```

Comment:
Looks like you accidentally included this file.

Reply:
It was me 😆 I just deleted it in the last commit.